pkg/queryfrontend/config.go

```diff
@@ -27,6 +27,21 @@ const (
 	MEMCACHED ResponseCacheProvider = "MEMCACHED"
 )
 
+var (
+	defaultMemcachedConfig = MemcachedResponseCacheConfig{
```

Comment:
Since the default config is available now, should the e2e tests change as well to not duplicate the configuration?

Reply:
Hi, reusing the default config is a good idea. But I think we want this default config private, so we have to duplicate a little bit.

Reply:
That's fair! SGTM